networkx/algorithms/tests/test_cluster.py

```diff
@@ -2,6 +2,8 @@
 
 import networkx as nx
 
+from ..cluster import all_triangles
+
```

Comment:
I think this won't be necessary anymore once the new function is properly exposed in the top-level namespace. You'll be able to just call `nx.all_triangles` then.